.github/workflows/welcome-first-time-contrib.yml

```diff
@@ -0,0 +1,18 @@
+name: Greeting the new contributor 
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```suggestion
        repo-token: ${{ secrets.AUTH_TOKEN }}
```

Reply:
This change is causing the action to fail. 
![image](https://github.com/json-schema-org/website/assets/63534268/ee72d401-b7bd-4585-83d2-4cc5aed79fc5)

Reply:
> This change is causing the action to fail. 

I shared with you the name of the token in our org. If you are executing it in your repo you'll need to rename the secret to use the same than ours, otherwise this will fail in production.

Reply:
Okay, Got it!